src/VAERSFileDownloader.py

```diff
@@ -16,7 +16,7 @@
 def solveCaptchaAndStartFileDownload(driver, captchaImageFile):
     saveCaptchaImageAs(driver, captchaImageFile)
     textInCaptchaImage = getTextInCaptchaImage(captchaImageFile)
-    display('textInCaptchaImage: ', textInCaptchaImage)
+    print('textInCaptchaImage: ', textInCaptchaImage)
```

Comment:
remove space after colon: print('textInCaptchaImage:', textInCaptchaImage)

Reply:
okay sure that sounds good